### ISSUE TITLE
MultiMiner / BFGMiner Offline install

### ISSUE THREAD

**Issue by zunami** [nwoolls/MultiMiner#383]
i have already read that installing the bfgminer is a problem under windows. 

now the browser and firewalls are blocking the bfgminer. 
every time i start the multiminer an error occurs regarding the download of the bfgminer and causes an error because no connection can be created.

now i was thinking i could download the bfgminer myself. the bfgminer is not a program with a fixed installation directory. 

the problem i have is that there is no manual. neither on the internet nor in the wiki. where i put the BFGMiner tool so that the multiminer recognizes it and wants to install it constantly.

**Comment by nwoolls** [comment]
Please see the following Wiki topic:

* https://github.com/nwoolls/MultiMiner/wiki/Backend-Miners

You place the miner(s) in the `MultiMiner\Miners\` directory. e.g. `MultiMiner\Miners\BFGmMiner\`.

**Comment by nJavaheri** [comment]
I have a similar problem. I am using Windows 10. 
1. I installed MultiMiner. On the first run it asked me if I wanted to download BFGMiner. I selected yes, but it couldn't proceed giving this error: "Value cannot be null. Parameter name: uriString"
2. Then I decided to download BFGMiner manually from here (the 64bit version): http://bfgminer.org/ 
I extracted the zipped files and placed them in "~MultiMiner\Miners\BFGMiner" and ran MultiMiner again. This time I immediately got this error: "Version string portion was too short or too long". 
3. If I rename the folder, e.g. to "~MultiMiner\Miners\bfgminer-5.5.0-win64" I get the error mentioned at 1, again. Like if it doesn't recognize the miner. 
Any suggestions?
